src/__global.zig

```diff
@@ -143,6 +143,9 @@ pub fn notimpl() noreturn {
 // Make sure we always print any leftover
 pub fn crash() noreturn {
     @setCold(true);
+    if (Environment.isDebug) {
+        @panic("Global.crash()");
```

Comment:
I think you want `@breakpoint` and not `@panic`.

Reply:
This was added to print a stack trace when a debug build is invoked without a debugger. This is a pattern i'm using in some places to make certain errors panic in debug, as tracing the message to the source code takes time and being able to click and see where the message is coming from is more productive. In a release, crash still acts as this "quiet panic"

- panic will also break on the callsite
- breakpoint will not print this info but break immediatly, or otherwise exit without a printed trace

Reply:
I still dont' think this should be done. It'll mess up tests and differs a lot from release build behavior. Can you add an environment variable that is disabled by default?

Reply:
that is reasonable.